src/binding-hash.h

```diff
@@ -48,11 +49,14 @@ class BindingHash : public std::unordered_multimap<std::string, Binding> {
 
   Index FindIndex(const Var&) const;
 
+  Index FindIndex(const string_view& name) const {
+    auto iter = find(name.to_string());
+    return iter != end() ? iter->second.index : kInvalidIndex;
+  }
+
   Index FindIndex(const StringSlice& name) const {
     auto iter = find(string_slice_to_string(name));
-    if (iter != end())
-      return iter->second.index;
-    return kInvalidIndex;
+    return iter != end() ? iter->second.index : kInvalidIndex;
   }
```

Comment:
Maybe consider adding a method

Index FindIndex(std::string name) const {
  auto iter  = find(name);
  return iter != end() ? iter->second.index : kInvalidIndex;
}

and then simplify the bodies of the other two FindIndex() methods to call this with the extracted string?